src/main/java/de/metas/ui/web/pickingV2/packageable/process/PackageablesView_PrintPicklist.java

```diff
@@ -108,19 +109,34 @@ protected String doIt()
 		return MSG_OK;
 	}
 
-	private void pickIfNeeded(final PackageableRow row)
+	private void createPickingCandidatesIfNeeded(final PackageableRow row)
 	{
 		final boolean existsPickingCandidates = pickingCandidateService.existsPickingCandidates(row.getShipmentScheduleIds());
 		if (!existsPickingCandidates)
 		{
-			final @NonNull PickingConfigV2 pickingConfig = PickingConfigV2.builder()
-					.createPickingCandidatesOnly(true)
-					.pickingReviewRequired(false)
-					.build();
-			productsToPickRowsRepository.pick(row, pickingConfig);
-
-			// commit in DB so that the report can access it
-			Services.get(ITrxManager.class).commit(getTrxName());
+
+			Services.get(ITrxManager.class).runInNewTrx(new TrxRunnable2()
```

Comment:
please restore the comment 
```
// commit in DB so that the report can access it
```
you can in an intermediate commit, because without it i was scratching my head about why you actually have this trx-manager-stuff